config/secrets.yml

```diff
@@ -31,6 +32,8 @@ test:
 
 production:
   <<: *production_and_staging
+  ga_tracking_id: UA-19524293-4
 
 staging:
   <<: *production_and_staging
+  ga_tracking_id: UA-19524293-3
```

Comment:
yes - the main goal is that any test data is kept away from production analytics; I am not that much concerned with keeping staging / dev / test together.